stock_by_warehouse_mrp_bom/views/mrp_bom_inherit.xml

```diff
@@ -0,0 +1,12 @@
+<odoo>
+    <record id="mrp_bom_view_form_inherit" model="ir.ui.view">
+        <field name="name">mrp.bom.stock.warehouse</field>
+        <field name="model">mrp.bom</field>
+        <field name="inherit_id" ref="mrp.mrp_bom_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='bom_line_ids']/tree" position="attributes">
+                <attribute name="editable"/>
```

Comment:
sorry, but I don't agree with this change. We should try to avoid UX changes unless necessary.

BTW, even if you needed this view:
- File name should be `mrp_bom_views.xml`
- External ID should be the same as the original one

For more info, you could check [the guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming)

Reply:
In the [original view](https://github.com/odoo/odoo/blob/38e5d002/addons/mrp/views/mrp_bom_views.xml#L82) the field is only editable in the tree view, the way this widget is presented right now needs the form view to show it

If this change is not done a refactor should be done to show the widget in the line, something like the one I did in ABSA (asynchronous)

Reply:
@CarmenMiranda but if the tree view is set to be inline editable, shouldn't the form view be displayed anyway when not editing the record?

Reply:
True, I don't remember right now if it can be triggered in edit mode, @agarzaarvizu Could you review if the button can be triggered please

Reply:
@CarmenMiranda If I'm in the BoM and click the line, the BoM changes to the edit mode automatically and the only button in the line is the one that shows you the product_product view to editing the view. I'm not sure if that's what you meant.

Reply:
OK, then I agree with this change. Besides, it's the same [done for sales in v14](https://github.com/Vauxoo/addons-vauxoo/blob/08280b001af01ca202d944369ace2f20e8e70efd/stock_by_warehouse_sale/views/sale_view.xml#L26)